vars/withPipeline.groovy

```diff
@@ -107,65 +106,42 @@ def call(type, String product, String component, Closure body) {
         }
 
         onPR {
-          onTerraformChangeInPR{
-            sectionDeployToEnvironment(
-              appPipelineConfig: pipelineConfig,
-              pipelineCallbacksRunner: callbacksRunner,
-              pipelineType: pipelineType,
-              subscription: subscription.nonProdName,
-              aksSubscription: aksSubscriptions.aat,
-              environment: environment.nonProdName,
-              product: product,
-              component: component,
-              tfPlanOnly: true
-            )
```

Comment:
so its very essential these lines stay in, we need a tf plan of aat (since its the staging environment) to run every time as a separate step no matter what the topic/label is

Reply:
done